pr Implements feature request #1887  rebase add osd

laz2wiringpi opened the pull request:
from 
delay for voltage warnings #6691

changed to vbat
Update Battery.md
Add Menu osd Alarm delay

mikeller commented on src/main/sensors/battery.c:
```diff
@@ -228,24 +230,35 @@ void batteryUpdatePresence(void)
         case BATTERY_CRITICAL:
             if (voltageMeter.filtered > batteryCriticalVoltage) {
                 voltageState = BATTERY_WARNING;
+                lastVoltageOkMillis = millis();
```
Using this here is very misleading, as the voltage is not 'ok'. Maybe call it `lastVoltageChangeMs`. (In general, variables used to keep milliseconds are called `...Ms` in Betaflight.

mikeller commented on src/main/sensors/battery.h:
```diff
@@ -63,6 +63,8 @@ typedef struct batteryConfig_s {
     uint8_t vbatLpfPeriod;                  // Period of the cutoff frequency for the Vbat filter (in 0.1 s)
     uint8_t ibatLpfPeriod;                  // Period of the cutoff frequency for the Ibat filter (in 0.1 s)
+    uint8_t vbatDurationForWarning;      // Period voltage has to sustain before the battery state is set to BATTERY_WARNING (in 0.1 s)
+    uint8_t vbatDurationForCrit;         // Period voltage has to sustain before the battery state is set to BATTERY_CRIT (in 0.1 s)
```
`vbatDurationForCritical`.

laz2wiringpi replied:
fixed should i rebase

laz2wiringpi commented:
squashed and rebase

laz2wiringpi commented:
@mikeller 
you had asked the org pr for a rebase delay for voltage warnings #6691
and as it was stale i had added/changed a few things  and some other suggestions 
delay for voltage warnings #6691

changed to vbat
Update Battery.md
Add Menu osd Alarm delay

vbat_duration_for_crit
batteryConfig_vbatDurationForCrit
to
 vbat_duration_for_critical
batteryConfig_vbatDurationForCritical

lastVoltageOkMillis rename

laz2wiringpi commented:
@mikeller thanks, Sorry about the confusion .
  should also close  
pr  #6691 
and the feature request #1887 ?
